src/connectors/archive.js

```diff
@@ -39,7 +39,29 @@ Connector.getArtistTrack = () => {
 		track = trackParts[2];
 	}
 
+	if (hasAllTracksNumericPrefix('.jwrowV2 .ttl')) {
+		track = track.trim().replace(/^(\d+\w+)/, '');
+	}
+
```

Comment:
Will this work if we move this code inside a MetadataFilter function?

Reply:
How would that work? what would it be like?

Reply:
```js
const filter = new MetadataFilter({ track: removeNumericPrefixes });

Connector.applyFilter(filter);

function removeNumericPrefixes(track) {
	if (hasAllTracksNumericPrefix('.jwrowV2 .ttl')) {
		return track.trim().replace(/^(\d+\w+)/, '');
	}

	return track;
}
```

Also, it's worth using a constant for `.jwrowV2 .ttl` selector.

Reply:
Thanks - I'll give this a shot later :pray: